src/lib/variants.ts

```diff
@@ -67,6 +67,16 @@ export interface Test {
     // they are always 0 and 1?
     audience?: number;
     audienceOffset?: number;
+
+    // These are specific to hardcoded tests
+    start?: string;
+    expiry?: string;
+    author?: string;
+    description?: string;
```

Comment:
@tjmw yeah it's interesting. I deliberately didn't port the entire test model from Frontend, as Contributions seemed to expose/use only a subset in their tooling and, arguably, the Frontend client-side framework is too flexible.

Reply:
Yeah, potentially we could jettison a lot of these: 

I.e.

    // these could all be left as comments instead of adding to the model
    // in general, I like metadata like this, but they don't exist in the 
    // contributions tools and aren't used anywhere
    description
    author
    successMeasure
    idealOutcome

    // I think we just ignore this (always 'All')?
    audienceCriteria

Reply:
The six metadata props in the hardcoded test have been deleted from the `Test` interface and refactored here as comments at the top of the file. There's no business logic around them, not on our side anyway.